src/module/system/action-macros/athletics/combat-maneuver.ts

```diff
@@ -0,0 +1,67 @@
+import type { ActorPF2e } from "@actor";
+import { SingleCheckActionUseOptions, SingleCheckActionVariant } from "@actor/actions/index.ts";
+import { ItemPF2e, WeaponPF2e } from "@item";
+import { WeaponTrait } from "@item/weapon/index.ts";
+import { ActionMacroHelpers } from "@system/action-macros/index.ts";
+import { CheckResultCallback } from "@system/action-macros/types.ts";
+
+interface CombatManeuverActionUseOptions {
+    /**
+     * The UUID or item instance of the weapon used to perform the disarm maneuver with, if any. Explicitly setting it
+     * to false will disable the automatic attempted lookup of a wielded weapon suitable to use for the action.
+     */
+    item: EmbeddedItemUUID | WeaponPF2e<ActorPF2e> | false;
+}
+
+function isOwnedItemUUID(uuid: EmbeddedItemUUID | string, actor: ActorPF2e) {
+    return uuid.match(/^Actor\.[a-zA-Z0-9]+\.Item\.[a-zA-Z0-9]+$/) && uuid.startsWith(`Actor.${actor.id}`);
+}
+
+abstract class CombatManeuverActionVariant extends SingleCheckActionVariant {
+    static resolveWieldedWeapon(
```

Comment:
I think the current trend for these functions is to have them as functions (not methods), and put them in a helpers.ts file somewhere. It does frustrate people since these helpers are not accessible outside of the pf2e project though when done that way, but it is consistent.

Reply:
Yep, and I agree with that approach. But as long as we need to maintain the old action functions we need to expose some stuff outside the action class implementations until we can remove them.